Makefile

```diff
@@ -152,7 +152,8 @@ fund-local-all: ## use faucet script to fund all the local identities
 		--network anvil-localhost \
 		--contracts-root "./ethereum/contracts" \
 		--identity-prefix "${id_prefix}" \
-		--identity-directory "${id_dir}"
+		--identity-directory "${id_dir}" \
+		--provider-url "http://localhost:8545"
```

Comment:
Why the rpc-provider url is using localhost? 
Why don't we use the production url: https://gnosis.rpc-provider.prod.hoprnet.link

Same in scripts/setup-local-cluster.sh changeset